dashboard/test/controllers/activities_controller_test.rb

```diff
@@ -898,7 +898,7 @@ def expect_controller_logs_milestone_regexp(regexp)
 
     # find localized test strings for custom lesson names in script
     assert response.key?('lesson_changing'), "No key 'lesson_changing' in response #{response.inspect}"
-    assert_equal('milestone-lesson-1', response['lesson_changing']['previous']['name'])
```

Comment:
After a bit of investigation, I still don't really understand exactly where the previous value of `milestone-lesson-1` was coming from. However, I do believe the new value added in this PR is the correct one, so I decided it wasn't worth getting all the way to the bottom of this.